docs_user/modules/proc_deploying-the-bare-metal-provisioning-service.adoc

```diff
@@ -173,8 +174,8 @@ spec:
 * After applying the configuration update to {rhos_acro}, apply the configuration and start the related services. The {bare_metal} begins to poll power state of the bare metal nodes:
 +
 ----
-openstack endpoint list |grep ironic
-openstack baremetal node list
+$ openstack endpoint list |grep ironic
+$ openstack baremetal node list
 ----
 +
-The time required for the {bare_metal} to review and reconcile the power state of bare metal nodes is dependent upon the number of operating conductors through the `replicas` parameter and which are present in the {bare_metal} deployment being adopted.
+The time required for the {bare_metal} to review and reconcile the power state of bare metal nodes depends on the number of operating conductors through the `replicas` parameter and which are present in the {bare_metal} deployment being adopted.
```

Comment:
I’m not sure how the text below these commands ("The time required…") relates to them. This text seems like it belongs in the introduction of the procedure, or in step 1. I’m also confused by the last part of the sentence “which are present in the {bare_metal} deployment being adopted”. Does "which" refer to the number of operating conductors?

Reply:
The Bare Metal service is periodicly checking the power status of baremetal nodes it is managing, it may take some time after service start to sync/check the status for all the baremetal nodes. It depends on the number of nodes, 10 - 50 - 200 nodes will not complete in the same time and if you have 1 conductor replica it will likely take longer than if you have 3 conductor replicas.

This was likely added there to ensure the user doing the adoption is not immediately alarmed if `openstack baremetal node list` output reports incorrect power status. If it does, give the system some time and re-run the node list command to see if the output "reconciles" with the actual state of the hardware being managed.

Reply:
@hjensas FYI ^

Reply:
ok, that makes sense.